x/gamm/pool-models/balancer/amm.go

```diff
@@ -234,7 +234,7 @@ func ensureDenomInPool(poolAssetsByDenom map[string]PoolAsset, tokensIn sdk.Coin
 	for _, coin := range tokensIn {
 		_, ok := poolAssetsByDenom[coin.Denom]
 		if !ok {
-			return sdkerrors.Wrapf(types.ErrDenomNotFoundInPool, invalidInputDenomsErrFormat, coin.Denom)
+			return types.ErrInputDenomsNotInPool
```

Comment:
Note to reviewer: reusing this error with stableswap doesn't allow for formatting a denom. Planning to create an issue to refactor stableswap side to improve this.